### PR TITLE
Feature/ds 86 update colors

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Review by literat — APPROVED** [review]

File: packages/web/src/components/Button/_theme.scss [hunk 1]
```diff
@@ -3,9 +3,10 @@
 $padding-x: tokens.$space-7;
 $padding-y: tokens.$space-4;
 $font-family: tokens.$font-family-default;
+$font-weight: tokens.$font-weight-semibold;
 $font-size: tokens.$font-size-base;
-$line-height: tokens.$line-height-default;
-$border-width: 1px;
+$line-height: tokens.$line-height-small;
```
Comment (adamkudrna) [inline packages/web/src/components/Button/_theme.scss:8]: According to Figma, buttons should be 40 px tall:

obrazek

vs.

obrazek

It seems that the sum of `line-height`, `border` and `padding` is smaller than we need…

**Review by adamkudrna — CHANGES_REQUESTED** [review]
All is fine except the height of buttons.

File: packages/web/src/components/Button/_theme.scss (hunk 1 above)
Comment (adamkudrna) [reply packages/web/src/components/Button/_theme.scss]: obrazek

🤔

Comment (adamkudrna) [reply packages/web/src/components/Button/_theme.scss]: Let's solve it in a separate PR, we cannot let it block the merge of other changes in this PR.

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**